test/functional/tools/output_filter_select.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="output_filter_select" name="output_filter_select" version="1.0.0">
+    <description>test for output filtering with selects</description>
+    <command><![CDATA[
+#if "A" in str($sel).split(",")
+    echo 'test' > A &&
+#end if
+#if "AA" in str($sel).split(",")
+    echo 'test' > AA &&
+#end if
+echo $sel 
+    ]]></command>
+    <inputs>
+        <param name="sel" type="select" multiple="true">
+            <option value="A">A</option>
+            <option value="AA">AA</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="A" format="txt" from_work_dir="A">
+            <filter>"A" in sel</filter>
+        </data>
+        <data name="AA" format="txt" from_work_dir="AA">
+            <filter>"AA" in sel</filter>
+        </data>
```

Comment:
The important part of this tool test is the output filters (sorry for the confusion).

When the filter is evaluated the content of `sel` is a string if only one item is selected (which leads to a failure in test 2, since `"AA"` also trigger that that outputs `"A"` and `"AA"` are included).

The command block only verifies that selects work as I expect them to work in cheetah.